src/pfe/portal/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "server.js",
   "scripts": {
     "start": "./npm-start.sh",
-    "prod": "node server.js",
-    "dev": "nodemon server.js",
+    "prod": "node --inspect=0.0.0.0:9777 server.js",
+    "dev": "node --inspect=0.0.0.0:9777 server.js",
```

Comment:
I'm assuming this is enabling the debugger. Do we want to leave debug on and available on a port in production? (Not sure if running node with `--inspect` has a performance impact.)